app/controllers/application_controller/timelines.rb

```diff
@@ -128,41 +128,7 @@ def tl_chooser
   end
 
   def getTLdata
```

Comment:
It was like this before, but why is it named in a funny case?  I see that we are adding routes for it now, and I think having camel case routes might not be the best idea.